backend/routes/root.js

```diff
@@ -9,7 +9,7 @@ router.get("/login", (_request, response) => {
   response.render("login.ejs");
 });
 
-router.get("/register", (_request, response) => {
+router.get("/user/register", (_request, response) => {
```

Comment:
Should this be just /register?
If so, needs to changed to /register in href in `home.ejs` as well

If not, sign up href in `login.ejs` needs to be updated

Reply:
I changed this for consistency purposes because the renders back to the register page from within `userController` (e.g [this](https://github.com/sfsu-csc-667-fall-2023/team-term-project-group-d/pull/26/files#diff-93c8daf456ea27cd69fcc8b29f1e5acc74b924db16877111991f765544cd9721R57)) were sending the client back to the `/user/register` url. I can definitely change this back if that is not what we want, I am not the most familiar with express.

Reply:
Ah, yeah. There's a lot of difficulty to send errors from backend to frontend in a clean way, I wonder if prof will allow us to use something like [this](https://www.npmjs.com/package/connect-flash) or [this](https://www.npmjs.com/package/express-flash)

I think we could also use sessions here as well, where you store the data (message: errors or successes) in the session. Since I think res.render('/path') goes through our app.use() again, it could be extracted from [req.session.errors or whatever] into [res.locals.errors or whatever (useable in Views)] and deleted from req.session so it doesnt persist for too long in the session's data.